CONTRIBUTORS.md

```diff
@@ -284,6 +284,7 @@ video tutorials.
  - Jonas Ådahl
  - Lasse Öörni
  - Leonard König
+ - Grzesiek11
```

Comment:
The contributors are normally alphabetic by last or only name, but I can fix that up after merge if needed.

Reply:
Looks like I'm already listed in the proper position, and this is just a duplicate coming from my error during merging back master.